src/query/service/src/servers/http/clickhouse_handler.rs

```diff
@@ -247,6 +246,16 @@ pub async fn clickhouse_handler_get(
             .set_batch_settings(&params.settings)
             .map_err(BadRequest)?;
 
+        if !settings
+            .get_enable_clickhouse_handler()
+            .map_err(InternalServerError)?
+        {
+            return Ok(Body::from_string(
+                "default settings: enable_clickhouse_handler is 0".to_string(),
+            )
+            .with_content_type("text/abc"));
```

Comment:
What's the `text/abc` mean?

Reply:
emmm, should `text/plain`? sorry I am not familiar with http protocol

Reply:
should return PoemError (instead of Ok) when not 
what about [405 Method Not Allowed](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405)